controllers/lists.js

```diff
@@ -0,0 +1,81 @@
+const {StatusCodes} = require('http-status-codes')
+const List = require('../models/Lists')
+const {BadRequestError, NotFoundError} = require('../errors');
+
+const getAllLists = async (req, res) => {
+    const lists = await List.find({createdBy: req.user.userId}).sort('createdAt');
+    res.status(StatusCodes.OK).json({lists, count: lists.length});
+}
+
+const getList = async (req, res) => {
+    const {user:{userId}, params: {id: listId}} = req; 
+    const list = await List.find({
```

Comment:
Need to use findOne here.  Otherwise, list is assign an empty list and the NotFoundError is not thrown